README.rst

```diff
@@ -12,13 +12,13 @@ collection of tools to this end.
 
 librascal is currently considered a standalone code. However, we aim to
 provide enough flexibility to interface it with other codes such as
-LAMMPS and PLUMED-2.0.  It can be used as a C++ library as well as a
-python module.  To be able to call it from python, we have used the
+LAMMPS and PLUMED-2.0. It can be used as a C++ library as well as a
```

Comment:
French spacing...? In all seriousness, though, the double space makes it easier to read in code; this is of course ignored by the typesetting to give the "modern" single-space.

Reply:
No, I have [proselint](https://github.com/amperser/proselint) configured locally, and it was complaining :smile: 

I though this was standard in English too, but I can change it back if you want.

Reply:
The exact message was "Inconsistent spacing after period (1 vs. 2 spaces)." so I guess both are accepted in English, and proselint just want me to be more consistent :smile: